docs/rfcs/0002/readme.md

```diff
@@ -84,6 +84,8 @@ spec:
                 mountPath: /foo/persistent
               - name: nnf-config
                 mountPath: /nnf/config
+              securityContext:
```

Comment:
Is secruityContext.allowPrivilegeEscalation defaulted to false? You can strip that if so

Reply:
I believe it's actually defaulted to `nil`: https://github.com/kubernetes/design-proposals-archive/blob/main/auth/no-new-privs.md#changes-of-securitycontext-objects. Or at least that's what I got from that documentation.

Reply:
nil is false

Reply:
I dont think that's true:

> This would allow users to set defaultAllowPrivilegeEscalation=false, overriding the default nil behavior of no_new_privs=false for containers whose uids are not 0.


allowPrivilegeEscalation setting | uid = 0 or unset | uid != 0 | privileged/CAP_SYS_ADMIN
-- | -- | -- | --
nil | no_new_privs=true | no_new_privs=false | no_new_privs=false
false | no_new_privs=true | no_new_privs=true | no_new_privs=false
true | no_new_privs=false | no_new_privs=false | no_new_privs=false

Reply:
I mean when SecruityContext is nil - AllowPrivilegeEscalation is false per https://github.com/kubernetes/kubernetes/blob/4d78db54a58e250b049c4fe17ac484e5c3ec662d/staging/src/k8s.io/pod-security-admission/policy/check_allowPrivilegeEscalation.go#L59-L79

Reply:
Ah, thank you for pointing that out. I'll remove it then.